chemprop/models/model.py

```diff
@@ -149,7 +149,7 @@ def training_step(self, batch: TrainingBatch, batch_idx):
         preds = self.predictor.train_step(Z)
         l = self.criterion(preds, targets, mask, w_s, self.w_t, lt_mask, gt_mask)
 
-        self.log("train/loss", l, prog_bar=True)
+        self.log("train_loss", l, prog_bar=True)
```

Comment:
What does this `self.log` function do?

Reply:
https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.core.LightningModule.html#lightning.pytorch.core.LightningModule.log

It passes information to the lightning progress bar to print. Right now we print something like `Epoch 0: 100%|███████████████████████████████████████████████████████████████| 1/1 [00:11<00:00, 11.95s/it, v_num=0, train/loss=1.250]`. The validation version uses `val_loss=...` instead as recommended by lightning.

Reply:
When we report the metrics, we use `"{val,test}/{metric}"`, so I think it makes sense for it to be `/` here, but instead change `val_loss` to `val/loss`? 
```
        self.log("train/loss", l, prog_bar=True)

        return l

    def validation_step(self, batch: TrainingBatch, batch_idx: int = 0):
        losses = self._evaluate_batch(batch)
        metric2loss = {f"val/{m.alias}": l for m, l in zip(self.metrics, losses)}

        self.log_dict(metric2loss, batch_size=len(batch[0]))
        self.log("val_loss", losses[0], batch_size=len(batch[0]), prog_bar=True)

    def test_step(self, batch: TrainingBatch, batch_idx: int = 0):
        losses = self._evaluate_batch(batch)
        metric2loss = {f"test/{m.alias}": l for m, l in zip(self.metrics, losses)}

        self.log_dict(metric2loss, batch_size=len(batch[0]))
```